WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/builders/ScanStateListItemsBuilder.kt

```diff
@@ -257,15 +257,24 @@ class ScanStateListItemsBuilder @Inject constructor(
         val clickableText = resourceProvider.getString(R.string.scan_here_to_help)
 
         val descriptionTextResId = if (threatsCount > 1) {
-            R.string.scan_idle_with_threats_description_plural
-        } else R.string.scan_idle_with_threats_description_singular
-        val descriptionText = htmlMessageUtils
-            .getHtmlMessageFromStringFormatResId(
-                descriptionTextResId,
-                "<b>$threatsCount</b>",
-                "<b>${site.name ?: resourceProvider.getString(R.string.scan_this_site)}</b>",
-                clickableText
-            )
+            R.string.scan_idle_threats_description_plural
+        } else R.string.scan_idle_threats_description_singular
+        val descriptionText = if (threatsCount > 1) {
+            htmlMessageUtils
+                    .getHtmlMessageFromStringFormatResId(
+                            descriptionTextResId,
+                            "<b>$threatsCount</b>",
+                            "<b>${site.name ?: resourceProvider.getString(R.string.scan_this_site)}</b>",
+                            clickableText
+                    )
+        } else {
+            htmlMessageUtils
+                    .getHtmlMessageFromStringFormatResId(
+                            descriptionTextResId,
+                            "<b>${site.name ?: resourceProvider.getString(R.string.scan_this_site)}</b>",
+                            clickableText
+                    )
+        }
```

Comment:
nit: now that we have different code paths for calling `getHtmlMesageFromStringFormatResId` anyway, probably not worth having an `if` branch just for the `descriptionTestResId` separately.

i.e. remove the `val descriptionTextResId = …` lines and directly use:
```kt
        val descriptionText = if (threatsCount > 1) {
            htmlMessageUtils
                    .getHtmlMessageFromStringFormatResId(
                            R.string.scan_idle_threats_description_plural,
                            "<b>$threatsCount</b>",
                            "<b>${site.name ?: resourceProvider.getString(R.string.scan_this_site)}</b>",
                            clickableText
                    )
        } else {
            htmlMessageUtils
                    .getHtmlMessageFromStringFormatResId(
                            R.string.scan_idle_threats_description_singular,
                            "<b>${site.name ?: resourceProvider.getString(R.string.scan_this_site)}</b>",
                            clickableText
                    )
        }
```

Reply:
Nice suggestion @AliSoftware 🌟, applied in 23da7f2.